spec/model/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                              bio: 'Teacher from Mexico.', post_counter: 0)
```

Comment:
- [ ] Some of your user tests were failing because you are defining the wrong attribute here. It should be `posts_counter`, not `post_counter`